functions.php

```diff
@@ -935,8 +935,8 @@ function async_post($url, $payload)  {
 }
 
 function sms_chunk_split($msg) {
-    $msg = preg_replace('/[\r\n]+/', ' ', $msg);
-    $chunks = wordwrap($msg, 1600, '\n');
+    $chunk_width = '1575';
```

Comment:
should be an integer instead of a string

Reply:
damn it i thought that